homeassistant/components/device_tracker/owntracks.py

```diff
@@ -36,6 +39,7 @@
 def setup_scanner(hass, config, see):
     """Setup an OwnTracks tracker."""
     max_gps_accuracy = config.get(CONF_MAX_GPS_ACCURACY)
+    waypoint_import_user = config.get(CONF_WAYPOINT_IMPORT_USER)
```

Comment:
To make this by default: `config.get(CONF_WAYPOINT_IMPORT_USER, True)`

We will need to do some sort of whitelisting/blacklisting functionality